osu.Game.Rulesets.Taiko/Judgements/TaikoJudgement.cs

```diff
@@ -19,13 +19,40 @@ protected override int NumericResultFor(HitResult result)
         {
             switch (result)
             {
-                default:
-                    return 0;
                 case HitResult.Good:
                     return 100;
                 case HitResult.Great:
                     return 300;
+                default:
+                    return 0;
             }
         }
+
+        /// <summary>
+        /// Retrieves the numeric health increase of a <see cref="HitResult"/>.
+        /// </summary>
+        /// <param name="result">The <see cref="HitResult"/> to find the numeric health increase for.</param>
+        /// <returns>The numeric health increase of <paramref name="result"/>.</returns>
+        protected virtual double HealthIncreaseFor(HitResult result)
```

Comment:
This should be moved to `Judgement.cs` as it should be used for all gamemodes (fine to leave unimplemented in other gamemodes for now).